tests/lib/eslint/eslint.js

```diff
@@ -143,8848 +147,9078 @@ describe("ESLint", () => {
         sinon.restore();
     });
 
-    after(() => {
-        shell.rm("-r", fixtureDir);
-    });
-
-    describe("ESLint constructor function", () => {
+    [
+        [],
+        ["unstable_config_lookup_from_file"]
+    ].forEach(flags => {
 
-        it("should have a static property indicating the configType being used", () => {
-            assert.strictEqual(ESLint.configType, "flat");
-        });
+        describe("ESLint constructor function", () => {
```

Comment:
Probably it's not necessary to repeat every test in this suite with and without the `unstable_config_lookup_from_file` flag. I understand the caution but many of these unit tests are not related to the config lookup.

Reply:
It's easier to do this than it is to try and identify which tests would actually be affected.